### ISSUE TITLE
Syntax error in Octave 6.0.1

### ISSUE THREAD

**Issue by sffc** [ltfat/ltfat#115]
Hello,

When installing `ltfat-2.3.1.tar.gz` in Octave built from the tip of the stable branch (currently version 6.0.1), I get the following error printed to the console.

```
error: parse error near line 153 of file /usr/local/share/octave/packages/ltfat-2.3.1/nonstatgab/nsdgt.m

  syntax error

>>>                                                           times,f(win_range,:),g{ii}(idx));
                                                                  ^
error: called from
    doc_cache_create>create_cache at line 116 column 20
    gen_doc_cache_in_dir>@<anonymous> at line 150 column 17
    doc_cache_create>gen_doc_cache_in_dir at line 151 column 9
    doc_cache_create at line 62 column 12
    install>generate_lookfor_cache at line 820 column 5
    install at line 235 column 7
    pkg at line 566 column 9
```

The problematic line of code appears to be here:

https://github.com/ltfat/ltfat/blob/5f3e46bc82086c42d44207a522fd3f4723844060/nonstatgab/nsdgt.m#L128

I will take a look to see if it is possible to fix this.

**Comment by RSChegani** [comment]
This issue still exists for the latest version.

**Comment by allthatsounds** [comment]
Apologies, this is because there has not yet been a new LTFAT release due to changes in the core developer team. The release of LTFAT 2.5 is planned for the beginning of summer.

**Comment by sffc** [comment]
See the fix in my PR #116. It should work if you download and install ltfat from GitHub instead of from Octave Forge, or if you download it from Octave Forge and then apply the patch in #116.
